include/zephyr/logging/log_multidomain_helper.h

```diff
@@ -87,14 +87,14 @@ struct log_multidomain_source_cnt {
 /** @brief Content of the domain name message. */
 struct log_multidomain_domain_name {
 	uint8_t domain_id;
-	char name[0];
+	FLEXIBLE_ARRAY_DECLARE(uint8_t, name);
 } __packed;
 
 /** @brief Content of the source name message. */
 struct log_multidomain_source_name {
 	uint8_t domain_id;
 	uint16_t source_id;
-	char name[0];
+	FLEXIBLE_ARRAY_DECLARE(uint8_t, name);
```

Comment:
Should this and the others here be `char, name`?
```suggestion
	FLEXIBLE_ARRAY_DECLARE(char, name);
```

Reply:
thanks !